LICENSE

```diff
@@ -200,3 +200,28 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+This product bundles the following source code which is available under the listed license:
```

Comment:
I don't think markdown is really what we want here to list our dependencies' licenses. That is probably better served in a separate `3rdPartyLicenses.md` file or something

Reply:
@apiri what do you think?